test/conversion.jl

```diff
@@ -192,13 +227,32 @@ end
     @test GAP.julia_to_gap([1,"foo",BigInt(2)]) == x
     x = GAP.EvalString("[[1,2],[3,4]]")
     @test GAP.julia_to_gap([ 1 2 ; 3 4 ]) == x
-    
+
+    ## BitArrays
+    x = GAP.EvalString("BlistList([1,2],[1])")
+    y = GAP.julia_to_gap([true,false])
+    @test y == x
+    @test GAP.gap_to_julia( GAP.Globals.TNAM_OBJ( y ) ) == "list (boolean)"
+
     ## Tuples
     x = GAP.EvalString("[1,\"foo\",2]")
     @test GAP.julia_to_gap((1,"foo",2),Val(true)) == x
     x = GAP.EvalString("[1,JuliaEvalString(\"\\\"foo\\\"\"),2]")
     @test GAP.julia_to_gap((1,"foo",2)) == x
-    
+
+    ## Ranges
+    r = GAP.EvalString( "[]" )
+    @test GAP.julia_to_gap( 1:0 ) == r
+    @test GAP.julia_to_gap( 1:1:0 ) == r
+    r = GAP.EvalString( "[ 1 ]" )
+    @test GAP.julia_to_gap( 1:1 ) == r
+    @test GAP.julia_to_gap( 1:1:1 ) == r
+    r = GAP.EvalString( "[ 4 .. 13 ]" )
+    @test GAP.julia_to_gap( 4:13 ) == r
+    @test GAP.julia_to_gap( 4:1:13 ) == r
+    r = GAP.EvalString( "[ 1, 4 .. 10 ]" )
+    @test GAP.julia_to_gap( 1:3:10 ) == r
```

Comment:
I assume trying to convert `1:2^62` will throw an error? Perhaps this should also be tested for?